content/post/kubestash-v2024.6.4/index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Introducing KubeStash v2024.6.4
+date: "2024-06-05"
+weight: 10
+authors:
+- Md Ishtiaq Islam
+tags:
+- backup
+- disaster-recovery
+- kubernetes
+- kubestash
+- manifest-backup
+- manifest-restore
+- restore
+---
+
+We are pleased to announce the release of [KubeStash v2024.6.4](https://kubestash.com/docs/v2024.6.4/setup/), packed with new features and important bug fixes. You can check out the full changelog [HERE](https://github.com/kubestash/CHANGELOG/blob/master/releases/v2024.6.4/README.md). In this post, we'll highlight the key updates.
+
+### New Features
+
+Here, we are going to highlight the new features that have been introduced in this release.
+
+#### Workload Manifest Backup & Restore
+
+We've introduced functionality to backup and restore workload manifests within the `workload-addon`. These tasks encompass backing up and restoring the manifests of the workload, including their associated volumes, service account (if utilized), and service (only the headless service employed in StatefulSet).
```

Comment:
```suggestion
We've introduced functionality to backup and restore workload (Deployment/StatefulSet/DaemonSet) manifests within the `workload-addon`. The new `manifest-backup` and `manifest-restore` tasks of the `workload-addon` enable backing up and restoring of workload manifests, including their associated volumes, service account (if used), and service (in case of StatefulSet).
```